apps/prairielearn/src/sync/course-db.ts

```diff
@@ -1,17 +1,27 @@
-// @ts-check
 import * as path from 'path';
-const _ = require('lodash');
+import _ = require('lodash');
 import * as fs from 'fs-extra';
 import * as async from 'async';
 import * as jju from 'jju';
-import Ajv from 'ajv';
+import Ajv, { type JSONSchemaType } from 'ajv';
 import betterAjvErrors from 'better-ajv-errors';
 import { parseISO, isValid, isAfter, isFuture } from 'date-fns';
 
 import { chalk } from '../lib/chalk';
 import { config } from '../lib/config';
 import * as schemas from '../schemas';
-import * as infofile from './infofile';
+import {
+  hasErrors,
+  hasErrorsOrWarnings,
+  hasWarnings,
+  makeError,
+  type InfoFile,
+  addError,
+  makeInfoFile,
+  addWarning,
+  addErrors,
+  addWarnings,
+} from './infofile';
```

Comment:
Nit/opinion: I actually prefer the `import * as infofile` form in this context; `hasError`, `addWarnings`, etc. are all so vague without the `infofile` qualifier.

Reply:
Restored in 926f2d6bffd1c5880349e7d86179fb5dfcffeb70. I mainly wanted to avoid adding it to every reference to the `InfoFile` type, so I added an explicit type instead (which was the strategy used beforehand anyway).